packages/mobile/screens/SignUp.tsx

```diff
@@ -0,0 +1,244 @@
+        const error = graphQLErrors?.map((err: any) => err?.message);
+          setErrors({ server: error[0] });
+      });
+  };
+
+
+
+  return (
+    <Container>
+      <Header>
+        <Image resizeMode={'cover'}
+          source={require('../assets/logo-lithium.png')}
+        />
+        <WelcomeText>{'Welcome to Lithium KB'}</WelcomeText>
+      </Header>
+      <Title>{'Sign up'}</Title>
+      <Formik
+        initialValues={{ username: '', email: '', password: '', confirmation: '' }}
+        validationSchema={signupSchema}
+        onSubmit={submition}>
+
+        {({ values: { username, email, password, confirmation }, handleChange, handleSubmit, errors, touched, handleBlur }: { values: any, handleChange: any, handleSubmit: any, errors: any, touched: any, handleBlur: any }) => (
+          <React.Fragment>
+
+            {touched.username && errors?.username?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.username}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={username}
+                placeholder="Name..."
+                placeholderTextColor="#003f5c"
+                error={touched["username"] && errors["username"]?.length > 0}
+                onBlur={handleBlur('username')}
+                onChangeText={handleChange('username')} />
+                
+            </Input>
+
+            {touched.email && errors?.email?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.email}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={email}
+                placeholder="Email..."
+                placeholderTextColor="#003f5c"
+                error={touched["email"] && errors["email"]?.length > 0}
+                onBlur={handleBlur('email')}
+                onChangeText={handleChange('email')} />
+            </Input>
+
+            {touched.password && errors?.password?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.password}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={password}
+                secureTextEntry
+                placeholder="Password..."
+                placeholderTextColor="#003f5c"
+                error={touched["password"] && errors["password"]?.length > 0 }
+                onBlur={handleBlur('password')}
+                onChangeText={handleChange('password')} />
+            </Input>
+
+            {touched.confirmation && errors?.confirmation?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.confirmation}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={confirmation}
+                secureTextEntry
+                placeholder="Confirmation..."
```

Comment:
This isnt the appropriate placeholder

Reply:
????

Reply:
if you comment that puts the right placeholder at least

Reply:
As I already [stated before](https://github.com/Lithium-Software/workspace-library/pull/15#discussion_r408911903), please check the designs